scripts/lib/CIME/SystemTests/pgn.py

```diff
@@ -44,8 +44,7 @@
 FCLD_NC = 'cam.h0.cloud.nc'
 INIT_COND_FILE_TEMPLATE = \
     "SMS_Ly5.ne4_ne4.FC5AV1C-04P2.eos_intel.ne45y.{}.{}.0002-{:02d}-01-00000.nc"
-# FIXME: should 'cam' be 'atm' now?
-INSTANCE_FILE_TEMPLATE = '{}cam_{:04d}.h0.0001-01-01-00000{}.nc'
+INSTANCE_FILE_TEMPLATE = '{}eam_{:04d}.h0.0001-01-01-00000{}.nc'
```

Comment:
This is a file name template which still uses `cam` as it was created in the older version of E3SM. This should not be changed otherwise the test won't be able to find the files.

Reply:
> 
> 
> Wherever the code refers to "eam" or "elm", add some logic to possibly set that to "clm" or "cam" so that CESM could also use these tests.
> 
> You could use case.get_value("MODEL") which will return "e3sm" or "cesm".

As @singhbalwinder mentioned above, the current baseline is "cam" for these tests in E3SM. Should I add a switch for that as well so the tests can be compared across model versions? Something as simple as:

```python
if any(["cam" in _ for _ in os.listdir(base_dir)]):
    baseline_component = "cam"
else:
    baseline_component = "eam"
```

Reply:
@singhbalwinder a new baseline has to be created and it should use eam histories.  One reason we stopped running this test was because there WAS a climate changing PR and no one knew how to update the baseline.

Reply:
Ok, I will create the baselines with eam history files. I am currently looking into this test and will find out how to update baselines. After that, I will try reviving it in our test suites again.

Reply:
Sorry, I misread the code. This change looks good to me. That is, it is okay to change this string to `eam` vs. `cam` based on the host model (E3SM or CESM).

The code which actually forms the initial condition file name is on lines [93](https://github.com/ESMCI/cime/pull/3967/files#diff-885e3f4da13b63c9bb4a447c2b313c11a37fc0ec731992b555cde2c7d8eed615R93) and [94](https://github.com/ESMCI/cime/pull/3967/files#diff-885e3f4da13b63c9bb4a447c2b313c11a37fc0ec731992b555cde2c7d8eed615R94). These lines are not modified in this PR, so we should be good with this change.